lib/improver/tests/convection/test_DiagnoseConvectivePrecipitation.py

```diff
@@ -281,36 +243,7 @@ def test_multiple_lead_times_neighbourhooding(self):
             radii, lead_times=lead_times
             )._calculate_convective_ratio(cubelist, self.threshold_list)
         self.assertIsInstance(result, iris.cube.Cube)
-        self.assertArrayAlmostEqual(result.data, expected)
-
-    def test_specify_ens_factor(self):
-        """Test an example where the ens_factor is specified."""
-        expected = np.array(
-            [[[[0., 0., 0., 0.],
-               [0., 0., 0., 0.],
-               [0., 0., 0., 0.],
-               [0., 0., 0., 0.]]],
-             [[[0.25, 0.166667, 0.166667, 0.],
-               [0.166667, 0.11111111, 0.11111111, 0.],
-               [0.166667, 0.11111111, 0.11111111, 0.],
-               [0., 0., 0., 0.]]]])
-        data = np.full((2, 1, 4, 4), 1.0 * mm_hr_to_m_s)
-        data[1, 0, 1, 1] = 20.0 * mm_hr_to_m_s
-        cube = set_up_cube(
-            data, "lwe_precipitation_rate", "m s-1",
-            realizations=np.array([0, 1]))
-        radii = 4000.0
-        ens_factor = 0.8
-        cubelist = lower_higher_threshold_cubelist(
-            cube.copy(), cube.copy(), self.lower_threshold,
-            self.higher_threshold)
-        result = DiagnoseConvectivePrecipitation(
-            self.lower_threshold, self.higher_threshold,
-            self.neighbourhood_method,
-            radii, ens_factor=ens_factor
-            )._calculate_convective_ratio(cubelist, self.threshold_list)
-        self.assertIsInstance(result, iris.cube.Cube)
-        self.assertArrayAlmostEqual(result.data, expected)
+#        self.assertArrayAlmostEqual(result.data, expected)
```

Comment:
I think we need to understand why this doesn't work if that is the case. Is this not just testing lead times? Why should removing ens_factor break this test?

Reply:
I accidentally hashed this out when I was testing other things and didn't put it back. I've changed it back now and it should work fine :)